nextflow_schema.json

```diff
@@ -489,13 +496,15 @@
                 },
                 "outdir_cache": {
                     "type": "string",
+                    "fa_icon": "fas fa-file",
                     "format": "directory-path",
                     "description": "The output directory where the cache will be saved. You have to use absolute paths to storage on Cloud infrastructure.",
                     "fa_icon": "fas fa-folder-open",
                     "hidden": true
                 },
                 "vep_out_format": {
                     "type": "string",
+                    "fa_icon": "fas fa-file",
```

Comment:
```suggestion
```

to fix failing tests

Reply:
I don't this the failing tests are due to the fa_icon

Reply:
I see, I was thinking about the linting failure...

:facepalm: 

Nevermind me.